app/Resources/views/themes/tastefulplanet/fields/ezrichtext/custom_tags/factbox.html.twig

```diff
@@ -0,0 +1,10 @@
+<div class="factbox">
```

Comment:
We should use CSS BEM naming convention :

<div class="factbox">
  <div class="factbox-card">
    <div class="factbox-card__header"></div>
    <div class="factbox-card__body"></div>
  </div>
</div>

or 

<div class="factbox-card">
    <div class="factbox-card__header"></div>
    <div class="factbox-card__body"></div>
</div>